Header header
uint32 object_id
string object_type
string object_name
geometry_msgs/Point[] key_points
geometry_msgs/Pose object_pose
string[] object_attributes
byte[] generic_data
